pom.xml

```diff
@@ -42,8 +42,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>8</maven.compiler.source>
```

Comment:
The target should be 1.8 instead

Reply:
Ah, that makes sense. I actually spun up a gradle build file because I didn't have maven installed. Lesson learned. Thanks for noticing that. And sorry for the very belated response. I completely missed that you commented here and I only just circled back to this at work.